js/scripts.js

```diff
@@ -49,4 +49,111 @@ document.addEventListener('DOMContentLoaded', function() {
 			e.currentTarget.parentElement.parentElement.querySelector(".expertise-detail").classList.toggle('is-hidden');
 		});
 	}
-});
\ No newline at end of file
+});
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
@jilaga if you don't mind me asking if this is a modal for our event schedule image, why are getting the navbar elements? This seems to repeat line 4 of the original file

Reply:
@vasconsaurus yes, I created a modal for the image so the contents of the schedule can be viewed full screen. I am a bit confused, Do you think it's an error? I would appreciate your assistance here.

Reply:
At first glance, I think it's a bit odd, but I'm not sure. I'll copy your code over here and play with it a bit, if you don't mind 🙂 But I think I'll only be able to do that tomorrow

Reply:
> At first glance, I think it's a bit odd, but I'm not sure. I'll copy your code over here and play with it a bit, if you don't mind 🙂 But I think I'll only be able to do that tomorrow

Sure...go ahead 😄